dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/util/ExpressionDimensionItemUtils.java

```diff
@@ -92,6 +92,45 @@ else if ( uids.length > 0 )
         return baseDimensionalItemObjects;
     }
 
+    /**
+     * Provides expression validation
+     *
+     * @param manager {@link IdentifiableObjectManager} service for item
+     *        delivery
+     * @param expression or indicator of expression dimension item
+     * @return true when expression is valid
+     */
+    public static boolean validateExpressionItems( IdentifiableObjectManager manager, String expression )
```

Comment:
if this method returns a boolean, I would find more appropriate to name like
`isValidExpressionItems`

Reply:
done